openmetadata-ui/src/main/resources/ui/cypress.config.ts

```diff
@@ -29,9 +29,6 @@ export default defineConfig({
     setupNodeEvents(on, config) {
       return plugins(on, config);
     },
-    specPattern: [
-      'cypress/new-tests/**/*.spec.{js,jsx,ts,tsx}',
-      'cypress/e2e/**/*.{js,jsx,ts,tsx}',
-    ],
+    specPattern: ['cypress/e2e/**/*.{js,jsx,ts,tsx}'],
```

Comment:
dont we need to run test from new-tests folder?

Reply:
I have merge it with existing structure, it was a temp folder which is removed now